templates/provisioning_tmpl.j2

```diff
@@ -0,0 +1,5 @@
+provisioning:
```

Comment:
Since these templates are specifically for configuring services within docker, I would move them to under `deployments/docker/`